src/accelerate/commands/launch.py

```diff
@@ -503,6 +503,12 @@ def launch_command_parser(subparsers=None):
         "--fsdp_backward_prefetch_policy",
         default=None,
         type=str,
+        help="This argument is deprecated. Use `fsdp_backward_prefetch` instead.",
```

Comment:
Let’s add a part that it will be removed in two 0.x.0 versions please. (0.27.0 I think)

Reply:
Done.